tests/Plotly.NET.ImageExport.Tests/ImageExport.fs

```diff
@@ -40,13 +40,17 @@ let ``Image export tests`` =
             ptestAsync "Chart.toBase64PNGStringAsync" {
                 let testBase64PNG = readTestFilePlatformSpecific "TestBase64PNG.txt"
                 
-                let! actual = (Chart.Point([1.,1.]) |> Chart.toBase64PNGStringAsync())
+                let! actual = (Chart.Point([1.,1.]) |> Chart.toBase64PNGStringAsync() |> Async.AwaitTask)
 
                 return 
                     Expect.equal
                         actual
                         testBase64PNG
                         "Invalid base64 string for Chart.toBase64PNGStringAsync"
             }
+            testCase "Chart.toBase64JPGString" <| fun () ->
```

Comment:
This is a great idea, as we can at least test if the render engine finishes this way. could you add this for the other image formats as well? maybe rename the tests to `Chart.toBase64PNGStringAsync terminates` etc